utils/mock-data-loader/setup-deletion-audit-logs.ts

```diff
@@ -0,0 +1,94 @@
+import { ObjectId } from 'mongodb';
+import { MONGO_DB_COLLECTIONS } from '@ukef/dtfs2-common';
+import { getCollection, getConnection } from './database/database-client';
+import { logger } from './helpers/logger.helper';
+
+export const deleteDeletionAuditLogsCollection = async () => {
+  logger.info('Dropping deletion audit logs collection');
+  const collection = await getCollection(MONGO_DB_COLLECTIONS.DELETION_AUDIT_LOGS);
+
+  try {
+    await collection.drop();
+  } catch {
+    // This will throw an error if the collection doesn't exist
+  }
+};
+
+export const setupDeletionAuditLogsCollection = async () => {
+  logger.info('Setting up deletion audit logs collection');
+  const dbConnection = await getConnection();
+
+  await dbConnection.createCollection(MONGO_DB_COLLECTIONS.DELETION_AUDIT_LOGS, {
+    validator: {
+      $jsonSchema: {
+        bsonType: 'object',
+        title: 'deletion audit logs validation',
+        required: ['collectionName', 'deletedDocumentId', 'auditRecord'],
+        properties: {
+          collectionName: {
+            bsonType: 'string',
+            description: 'collectionName must be a string',
+          },
+          deletedDocumentId: {
+            bsonType: 'objectId',
+            description: 'deletedDocumentId must be an ObjectId',
+          },
+          expireAt: {
+            bsonType: 'date',
+            description: 'expireAt must be a BSON date',
+          },
+          auditRecord: {
+            bsonType: 'object',
+            description: 'auditRecord is required',
+            properties: {
```

Comment:
Its possible to add conditional validation - ensure only one value is not null. But That might be overkill